var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -251,8 +252,6 @@ def cmake_args(self):
 
             # hip support
             if "+rocm" in spec:
-                options.append("-DVTKm_NO_DEPRECATED_VIRTUAL:BOOL=ON")
```

Comment:
I think this option is still necessary for VTK-m 1.9 and before. (The CMake variable will be ignored for VTK-m 2.0 and later.) Perhaps this should be kept in with a comment that it has no effect for VTK-m 2.0 and above.

Reply:
It is redundant with the virtuals variant, and is currently a bug. Before this change it was possible to say `vtk-m+rocm+virtuals` which is incorrect. After this change the `virtuals` variant handles the virtual part of this and the `rocm` variant handles the HIP part without ambiguous overlap

Reply:
OK. I see now.